### DISCUSSION TITLE
Unable to export more than 5 requests via postman integration extension in burp community version

### DISCUSSION THREAD

**Discussion by Rachitas870** [community/community#32235] [Other Feature Feedback, Questions, & Ideas]
I am unable to export more than 5 requests via postman integration extension in burp community version.
Is there a limit in community version ?

**Comment by queenofcorgis** [comment]
Hi @Rachitas870,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Postman](https://learning.postman.com/docs/integrations/available-integrations/github/#troubleshooting-the-github-integration). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
